rectools/metrics/popularity.py

```diff
@@ -32,17 +34,23 @@ class AvgRecPopularity(MetricAtK):
     with this item.
 
     .. math::
-        ARP@k = \frac{1}{\left|U_{t}\right|}\sum_{u\in U_{t}^{}}\frac{\sum_{i\in L_{u}}\phi (i)}{\left| L_{u} \right |}
+        ARP@k = \frac{1}{|U_{t}|}\sum_{u\in U_{t}^{}}\frac{\sum_{i\in L_{u}}\phi (i)}{|L_{u}|}
+    .. math::
+        Normalized ARP@k = \frac{1}{|U_{t}|}\sum_{u\in U_{t}^{}}\frac{\sum_{i\in L_{u}}\frac{\phi(i)}{|I|}}{|L_{u}|}
```

Comment:
Formula is incorrect. Normalization is done not on the number of items |I| (which is actually length of the catalog). But on the sum of phi(i) for every item in catalogue (which is total number of interactions).
Please write it in a clear way